commands/apps_test.go

```diff
@@ -45,3 +48,86 @@ func TestCommandsAppsCreateAttemptsToValidateStackOnError(t *testing.T) {
 	assert.Error(err)
 	assert.Regexp("bad request: unable to find stack", err)
 }
+
+// Looks like nested functions are not yet supported by the compiler
+func OverwriteFile(loc string, data string) (err error) {
+	err = os.Remove(loc)
```

Comment:
Do you need to remove and recreate a file in order to rewrite it?

Reply:
good note, no I don't.